src/main/java/redis/clients/jedis/Client.java

```diff
@@ -689,6 +690,11 @@ public void zcount(final String key, final String min, final String max) {
     zcount(SafeEncoder.encode(key), SafeEncoder.encode(min), SafeEncoder.encode(max));
   }
 
+  @Override
+  public void zcount(final String key, final Endpoint<Double> min, final Endpoint<Double> max) {
+    zcount(SafeEncoder.encode(key), min.getRaw(), max.getRaw());
```

Comment:
Does this work if I do `Endpoint.of(Double.POSITIVE_INFINITY, true)`? (Specifically I think this will translate it to `(+inf`, does this work in Redis given that `+inf` is a special value?)

Reply:
@mina-asham works in redis-cli
```
ZCOUNT key (-inf (+inf
(integer) 0
```